.github/workflows/qa.yml

```diff
@@ -23,4 +23,23 @@ jobs:
           if [[ "" != "$SYMLINKS" ]]; then
             echo "::error::Symlinks are not allowed"
             echo "Found $SYMLINKS"
+            exit 1
+          fi
+
+      - name: No .gitkeep, use .gitignore
```

Comment:
Currently, each step runs only if the previous one is a success, meaning that we don't report all errors to the contributor at once. there is 2 ways to check them all:

1. use separate jobs
2. change the conditions to run the steps. see https://github.com/symfony/demo/blob/7ed6c4a801eb074ad4052563cf54bf0e380de816/.github/workflows/lint.yaml#L64 for an example (this requires setting an id on the last step of the setup, to be able to reference it for the one that should have succeeded before running the linting step)